README.md

```diff
@@ -61,11 +61,28 @@ cd scripts/
 python xpcs_reproc_client.py
 ```
 
-### Theta Configuration
+### ALCF Configuration
 
 Hopefully, this document is a little outdated and you're executing on Polaris!
 Please add, update, or correct information as things change. 
 
+## Environment Setup
+
+```
+  conda create -n gladier-xpcs
+  conda activate gladier-xpcs
+  
+  pip install funcx
```

Comment:
funcx will be installed as dependency from Gladier, so we can leave this one off.